src/routes/docs/tooling/command-line/teams/+page.markdoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: article
+title: Deploy teams with CLI
+description: Efficiently deploy your Appwrite teams using the Command-Line Tool (CLI). 
+---
+
+{% info title="Before proceeding" %}
+Ensure you [**install**](/docs/tooling/command-line/installation#Installation) the CLI, [**login**](/docs/tooling/command-line/installation#get-started) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#Initialization) your Appwrite project.
+{% /info %}
```

Comment:
We have a way to create partials/tags to reduce duplicate code. Can that be used for this blog?